robot_name: mod6r_standard
offset_length: 0

dh_config:
 - 1.57 0 175 0 H
 - 0 300 0 0 H
 - 0 300 0 0 H
 - 1.57 0 175 0 L
 - -1.57 0 0 0 L
 - 0 0 256 0 L


## Examples 
wrist_unconvent:
 - 0.45 0 0 0 H
 - 1.00 0 0 0 H
 - 0 0 0 0 L

wrist:
 - -1.57 0 175 0 H
 - 1.57 0 0 0 L
 - 0 0 0 0 L

3R_unconvent:
 - 1.57 0 175 0 H
 - 0.45 300 0 0 H
 - 0 0 0 0 L

3R:
 - 1.57 0 175 0 H
 - 0 300 0 0 H
 - 0 0 0 0 L

2R:
 - 0 300 0 0 H
 - 0 300 0 0 L

6R:
 - 1.57 0 175 0 H
 - 0 300 0 0 H
 - 0 300 0 0 H
 - 1.57 0 175 0 L
 - -1.57 0 0 0 L
 - 0 0 256 0 L
 
6R_unconvent_0:
 - -1.207 0 175 0 H
 - 1.57 0 0 0 L
 - -2.272 300 0 0 L
 - 0.981 300 0 0 L
 - -2.453 0 0 0 L
 - 0 0 0 0 L
 
6R_unconvent_1:
 - 0.523 0 175 0 H
 - 0.01 0 0 0 L
 - -3.14 0 0 0 L
 - 0 300 0 0 L
 - -3.14 0 0 0 L

